### PR TITLE
Unit tests for Ethereum sdk modules

### PR DESCRIPTION
**Pull request by rranjan3**
Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>

### CODE REVIEW

File: sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    def test_work_order_complete_error(self):
+        read_json = read_json_file(
+            "work_order_get_result_error.json", ["./"])
+        wo_id = urandom(32).hex()
+
+        result = self.__eth_conn.work_order_complete(wo_id,
+                                                     read_json)
+        self.assertEqual(
+            result, 0, "Work order result submission should succeed")
```
Comment (Ram-srini) [inline sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py]: Is the assert message correct?

File: sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    def test_is_wo_id_in_event_positive(self):
+        response_event = {"args": {"workOrderId": b'g7V\xc7A',
+                                   "workOrderResponse":
+                                   '{"result":{"workloadId":"6563686f2",'
```
Comment (Ram-srini) [inline sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py]: Why these values are hard coded? Is it a mock function? Please add comments in the function.

File: sdk/avalon_sdk/ethereum/unit_tests/work_order_get_result.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+                "dataHash": "7A558709FBAF72D6CEEFD9AAA285989E852944E3D0A91D7A9409CCB3A2A5E056",
+                "data": "ER+fXUkeTNIWOyISiYeJUXF1IBXY4VmJanntmllr",
+                "encryptedDataEncryptionKey": "",
+                "iv": ""
+            }
+        ]
+    },
+    "id": 11140,
+    "jsonrpc": "2.0"
+}
```
Comment (Ram-srini) [inline sdk/avalon_sdk/ethereum/unit_tests/work_order_get_result.json]: Add a new line

File: sdk/avalon_sdk/ethereum/unit_tests/work_order_req.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "encryptedRequestHash": "5934CB58F73305FC7432FB5AA5753A547FAC79BDA2330DF6D1BD466624A7D4D2329FEBD1AB533AE6DB78A7F767DA493F",
+        "requesterSignature": "",
+        "inData": [
+            {
+                "index": 0,
+                "data": "VTq48Uu3gy+Zlge3mUeIRegNfF5CCw=="
+            }
+        ]
+    }
+}
```
Comment (Ram-srini) [inline sdk/avalon_sdk/ethereum/unit_tests/work_order_req.json]: Add a new line

File: sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py (hunk 1 above)
Comment (rranjan3) [reply sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py]: Yes @Ram-srini . The work_order_complete function should successfully write the result(execution result or error in execution) to the blockchain.

File: sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py (hunk 2 above)
Comment (rranjan3) [reply sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py]: Updated with comment.

File: sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+        wo_id = urandom(32).hex()
+
+        result = self.__eth_conn.work_order_complete(wo_id,
+                                                     read_json)
+        self.assertEqual(
+            result, 0, "Work order result submission should succeed")
+
+    def test_is_wo_id_in_event_positive(self):
+        """
+        This case mocks an event and veries wo_id_in_event function
```
Comment (danintel) [inline sdk/avalon_sdk/ethereum/unit_tests/test_ethereum_work_order_impl.py]: Change "veries wo_id_in_event function for a positive result"
to "verifies the wo_id_in_event function returns a positive result."

File: sdk/avalon_sdk/ethereum/unit_tests/work_order_req.json (hunk 4 above)
Comment (rranjan3) [reply sdk/avalon_sdk/ethereum/unit_tests/work_order_req.json]: Updated @Ram-srini . Could you please take a look again and merge if things are fine.

**Pull request merged**